### PR TITLE
fixed #1 skeleton written

### LINKED ISSUE #1: skeleton
skeleton

### PR DESCRIPTION
**Pull request by galgyeony**
basic skeleton.

@ETRIkimkiho please review

### CODE REVIEW

File: main.c [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#include <stdio.h>
+
+int add(int,int);
+int sub(int,int);
+int div(int,int);
+int multi(int,int);
+
+int main(int argc, char* argv[])
+{
+	int a,b,c;
```
Comment (galgyeony) [inline main.c]: please change variable names

File: main.c [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+	char operator;
+
+	printf("please enter an exp");
```
Comment (galgyeony) [inline main.c]: add colon and space at the end

**Comment by galgyeony** [comment]
:+1:

**Comment by ETRIkimkiho** [comment]
:+1:

**Pull request merged**